test/files/mock-insights

```diff
@@ -133,25 +134,19 @@ class handler(BaseHTTPRequestHandler):
 
 
 def insights_server(port):
+    # Let's put the certs into /var/lib/insights so that SELinux
+    # allows insights-client to actually read ca.crt when running as a
+    # systemd service.
+    subprocess.check_call("D=/var/lib/insights/mock-certs; [ -d $D ] || ( mkdir $D && cd $D && sscg )", shell=True)
```

Comment:
a more pythonic (and less shell-ish) version of this would be nicer ;)

Reply:
Agreed!

Reply:
Done.